easy/17.js

```diff
@@ -1,5 +1,18 @@
 function firstVowel(str) {
-	
+    str = str.toLowerCase();

```

Comment:
You can use findIndex

Reply:
i use search. findIndex doesn't work on string object. and convert it, imo, doesn't have sense .